python/tests/stress/arcticdb/version_store/test_mem_leaks.py

```diff
@@ -182,16 +310,61 @@ def proc_to_examine():
          run the test from command line again to assure it runs ok before commit 
 
     """
-    max_mem_bytes = 450_623_040
+    max_mem_bytes = 295_623_040
+
+    check_process_memory_leaks(proc_to_examine, 20, max_mem_bytes, 80.0)
+
+@pytest.mark.skipif(WINDOWS, reason="Not enough storage on Windows runners")
+@pytest.mark.skipif(MACOS, reason="Problem on MacOs")
+@SKIP_CONDA_MARK # Conda CI runner doesn't have enough storage to perform these stress tests
+def test_mem_leak_querybuilder_standard(arctic_library_lmdb):
+    """
+        This test uses old approach with iterations.
+        It is created for comparison with the new approach
+        with memray
+        (If memray is good in future we could drop the old approach)
+    """
+    lib: Library = arctic_library_lmdb
+
+    df = construct_df_querybuilder_tests(size=2000000)
+    size = df.shape[0]
+    start_date = df.iloc[0].name
+    end_date = df.iloc[0].name
+
+    symbol = "test"
+    lib.write(symbol, df)
+    del df
+    gc.collect()
+
+    queries = [query_group(), query_group_bigger_result(), query_apply(random_string(10)), query_resample()]
+
+    def proc_to_examine():
+        queries = [query_group(), 
+                    query_group_bigger_result(), 
+                    query_apply(random_string(10)), 
+                    query_resample(),
+                    query_row_range(size),
+                    query_date_range(start_date, end_date)]
```

Comment:
Because of the randomness involved, the memory usage of `query_row_range` and `query_date_range` will not be stable. They could be modified to return a fixed percentage of the input dataframe to solve this.

Reply:
makes sense - I will modify the code to have only start row/date random between first 40% of the rows and then return some persenatge of rows approx 57 for instance